src/Texture2D.js

```diff
@@ -117,8 +117,8 @@ var Texture2D = Texture.extend(function () {
             for (var i = 0; i < this.mipmaps.length; i++) {
                 var mipmap = this.mipmaps[i];
                 this._updateTextureData(_gl, mipmap, i, width, height, glFormat, glType, false);
-                width /= 2;
-                height /= 2;
+                width = Math.max(width / 2);
+                height = Math.max(height / 2);
             }
```

Comment:
I think it should be `width = Math.max(width / 2, 1)` ?

Reply:
So silly. Fixed.